app/src/main/java/org/fossasia/openevent/api/processor/SessionListResponseProcessor.java

```diff
@@ -13,6 +20,36 @@
     @Override
     public void success(SessionResponseList sessionResponseList, Response response) {
 
+        ArrayList<String> queries = new ArrayList<String>();
+
+        for (Session session : sessionResponseList.sessions) {
+            int id = session.getId();
+            String title = session.getTitle();
+            String subtitle = session.getSubtitle();
+            String summary = session.getSummary();
+            String description = session.getDescription();
+            String startTime = session.getStartTime();
+            String endTime = session.getEndTime();
+            String type = session.getType();
+            int track = session.getTrack();
+            String speakers = session.getSpeakers();
+            String level = session.getLevel();
+            int microlocation = session.getMicrolocation();
+            Session temp = null;
+            try {
+                temp = new Session(id, title, subtitle, summary, description, startTime, endTime, type, track, speakers, level, microlocation);
```

Comment:
You have the object already, you can create the query directly right? All you need to do is call a function, your object has the encapsulated fields already!